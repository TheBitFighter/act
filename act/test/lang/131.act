deftype foo (bool t,f)
{
   spec {
    exclhi(t,f)
   }
   methods {
    set { 
     self := 0
    }
   }
}

 
defproc test ()
{ 
  DR a,y;
  chp {
  *[
      a.t := y.f
   ]
  }

}
