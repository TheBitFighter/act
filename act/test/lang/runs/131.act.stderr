ERROR: File `131.act', line 8, col 11
	self used as an identifier, but not defined in this scope.
